### DISCUSSION TITLE
url_base - allow multiple, any, or disable

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#11304] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

155 gigs

### Storage for /nsm

312 gigs

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I would like to disable the auth cookie **url_base** requirement, or be able to add multiple entries in order to allow functionality with NAT.  I understand that I could use the hostname or FQDN, but as with many other users in this forum I have unique environments that hinder names.  Having a url_base cookie is great for security, but it seems like a long standing request from the community to have the option to remove it when needed.  The SO web interface is the only system I've used that has this requirement as the default, which is understandable given it's access to network information, but can we please be given the ability to manually disable it?

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment]
url_base is used more than just the cookies. All of the SO components talk to each other using SSL certificates so the url_base is factored into that as well. We do add the IP addresses and the host names to all the certs and add the important services to the /etc/hosts file for all the SO infrastructure in case there is no DNS. IPs are no longer as important in 2.4 and all services can use host names which keep the trust layers working properly.  I think it boils down to if you want to use NAT, then you need to use name resolution of some kind.

**Reply by innovate-support** [reply]
Thanks for the detailed reply.  Can we add both hostname and IP?

**Reply by TOoSmOotH** [reply]
No. If NAT is involved you will want to use hostname. IP is really more for testing.
